### PR TITLE
Fetching licenses via URL

### PR DESCRIPTION
**Pull request by arthurdm**
As per the [request made](https://github.com/docker-library/official-images/pull/8854#issuecomment-705205538) in the official Docker repo, this PR changes the Dockerfiles to fetch the licenses instead of having a local copy of them in the build context.  The end result should be the exact same.

Signed-off-by: Arthur De Magalhaes <ademagalhaes@gmail.com>

### CODE REVIEW

**Comment by tianon** [comment]
Can you please clarify whether these are the licenses of things in the downloaded artifacts or whether it also includes licenses for software that's installed via the package manager?

Also, is there some way this download could be verified? (https://github.com/docker-library/official-images#image-build)

**Comment by arthurdm** [comment]
hi @tianon - the licenses also cover things in the image outside of the WLP binary.

**Comment by tianon** [comment]
So those licenses should be collected/generated at build time instead so they don't become stale/bitrot, right? (or document how users can collect them themselves)

**Comment by arthurdm** [comment]
@tianon - the new PR pulls in the licenses during our image build time.  We need to include the licenses in the image to pass Red Hat certification.

File: ga/20.0.0.10/kernel/Dockerfile.ubi.adoptopenjdk11 [hunk 1]
```diff
@@ -41,6 +41,10 @@ RUN yum -y install shadow-utils unzip wget findutils openssl \
     && useradd -u 1001 -r -g 0 -s /usr/sbin/nologin default \
     && LIBERTY_URL=${LIBERTY_URL:-$(wget -q -O - https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/index.yml  | grep $LIBERTY_VERSION -A 6 | sed -n 's/\s*kernel:\s//p' | tr -d '\r' )}  \
     && wget $DOWNLOAD_OPTIONS $LIBERTY_URL -U UA-IBM-WebSphere-Liberty-Docker -O /tmp/wlp.zip \
+    && LICENSE_BASE=$(wget -q -O - https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/index.yml  | grep $LIBERTY_VERSION -A 6 | sed -n 's/\s*license:\s//p' | sed 's/\(.*\)\/.*/\1\//' | tr -d '\r') \
+    && wget ${LICENSE_BASE}en.html -U UA-IBM-WebSphere-Liberty-Docker -O /licenses/en.html \
+    && wget ${LICENSE_BASE}non_ibm_license.html -U UA-IBM-WebSphere-Liberty-Docker -O /licenses/non_ibm_license.html \
+    && wget ${LICENSE_BASE}notices.html -U UA-IBM-WebSphere-Liberty-Docker -O /licenses/notices.html \
```
Comment (yosifkit) [inline ga/20.0.0.10/kernel/Dockerfile.ubi.adoptopenjdk11:47]: Thanks for making this update. :+1: :heart:

A minor comment for future improvement: the [Security->Image Build section](https://github.com/docker-library/official-images/blob/c6c9d7b0b3a9a2edbcfd405a5ceed26c8d5c46ae/README.md#image-build) requires that remote artifacts be verified as much as possible, while this Dockerfile hasn't had checksums or gpg in the past, we do highly recommend that they are considered. (See the linked section for more explanation)

Comment (arthurdm) [reply ga/20.0.0.10/kernel/Dockerfile.ubi.adoptopenjdk11:47]: thanks @yosifkit / @tianon for the tips regarding the download verification.  I've opened an issue (https://github.com/WASdev/ci.docker/issues/364) to tackle this topic.

**Review by leochr — APPROVED** [review]
@arthurdm Looks good. Thanks.

**Review by wraschke — APPROVED** [review]

**Pull request merged**